source/collections/capped-collection.txt

```diff
@@ -0,0 +1,55 @@
+.. _capped-collection:
+
+==========================
+Create a Capped Collection
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:manual:`Capped Collections </core/capped-collections/>` are fixed sized 
```

Comment:
[suggestion]

I recommend using "fixed-size" to be consistent with this copy: https://www.mongodb.com/docs/manual/core/capped-collections/

Reply:
Done.